src/Http/Controllers/CP/Fields/FieldsController.php

```diff
@@ -190,6 +190,13 @@ protected function blueprint($blueprint)
                 'default' => 'visible',
                 'type' => 'select',
             ],
+            'localizable' => [
```

Comment:
Right now, this config field is showing on all fields, even if you're working on a single-site install or in managing fields in a Grid.

We should probably only be showing this option when the `canDefineLocalizable` prop is set to `true` (which is what we use [to determine if the localizable icon shows or not](https://github.com/statamic/cms/blob/897803d45fbb23da543d15fe7e5f0b496f5caa84/resources/js/components/blueprints/RegularField.vue#L19)).

Reply:
Ahh yeah, great catch. Don't think I can do that from the PHP array can I?

Reply:
Btw, something to consider. I wouldn't mind _always_ having the globe and this toggle available. I've had quite a few projects where I know it's gonna be multisite some day, but not yet. In those cases I would still like to prepare fields by setting them to localizable or not.

Reply:
> Ahh yeah, great catch. Don't think I can do that from the PHP array can I?

Nope, it'll need to be done in Vue.

Reply:
An if based on the handle? I could look into later.

Reply:
Unfortunately, that won't work for nested fields (like Grids), where the `canDefineLocalizable` is set to `false`. 

https://github.com/statamic/cms/blob/897803d45fbb23da543d15fe7e5f0b496f5caa84/resources/js/components/fieldtypes/grid/FieldsFieldtype.vue#L6

Ideally, we'd be using the same logic for this new field.

Reply:
I don't have a clue how to do this. Closing this one. If someone else wants to take a stab at it: be my guest :-).